### DISCUSSION TITLE
Internal server error: No known conditions for "./messages" entry in "@intlify/unplugin-vue-i18n" package

### DISCUSSION THREAD

**Discussion by rtarling** [intlify/bundle-tools#214] [Q&A]
Hi

I am migrating a Vue2 project to Vue3, and part of that is the i18n part.  I have translation JSON files in the src\locales direcrory.

As far as I can see I am following the steps correctly to set it up.  However I get the following message.  Does anyone know what is causing this, and how I can fix it?

11:26:26 [vite] Internal server error: No known conditions for "./messages" entry in "@intlify/unplugin-vue-i18n" package

**Comment by Sjoerd82** [comment]
Same here, in a Quasar project.

**Comment by ottopaulsen** [comment]
Did you figure out this? I am having the same problem in a Vue 3 project.

**Reply by Sjoerd82** [reply]
I have not...
